lib/tasks/link-audit.rake

```diff
@@ -0,0 +1,22 @@
+desc "Determine if all websites have links"
+task link_audit: :environment do
+  Redirection.find_each do |redirection|
+    url = URI(redirection.url)
+    html = Net::HTTP.get(url)
+    next_link = "hotlinewebring.club/#{redirection.slug}/next"
+    prev_link = "hotlinewebring.club/#{redirection.slug}/previous"
+
+    missing_links = []
+    missing_links << "next" if !html.include?(next_link)
+    missing_links << "prev" if !html.include?(prev_link)
+
+    if missing_links.empty?
+      print "\e[32m"
```

Comment:
Could you make a `green` function (or whatever color this is), like so?

```ruby
def green(text)
  print "\e[32m"
  puts "#{redirection.url} is all good"
  print "\e[0m"
end
```

I personally enjoy putting the "reset color" code in the function rather than printing it out after the `if`.